core/modules/server/routes/get-tiddlers-json.js

```diff
@@ -13,13 +13,15 @@ GET /recipes/default/tiddlers/tiddlers.json?filter=<filter>
 "use strict";
 
 var DEFAULT_FILTER = "[all[tiddlers]!is[system]sort[title]]";
+var DEFAULT_INCLUDE_NON_TIDDLERS = false;
 
 exports.method = "GET";
 
 exports.path = /^\/recipes\/default\/tiddlers.json$/;
 
 exports.handler = function(request,response,state) {
 	var filter = state.queryParameters.filter || DEFAULT_FILTER;
+	var include_non_tiddlers = state.queryParameters.include_non_tiddlers || DEFAULT_INCLUDE_NON_TIDDLERS;
```

Comment:
Surely we need to test that state.queryParameters.include_non_tiddlers is set to "yes"?

This is dangerous because if DEFAULT_INCLUDE_NON_TIDDLERS were true and state.queryParameters.include_non_tiddlers were false then the setting would end up being true. But as I say, I don't see the value in having a default fallback.

Reply:
You're right. If the line were `state.queryParameters.include_non_tiddlers || false` would this already resolve your concern, or do think it's still better to require `yes` as the positive value for the query parameter.

Reply:
We should test explicitly for the positive value "yes".